setup.py

```diff
@@ -1,13 +1,37 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
I'm not sure this `#!` line is needed, since it gets registered as a console script.